### PR TITLE
Shorten the error message in orphaned-objects monrun check

### PR DESCRIPTION
**Pull request by NikitaUnisikhin**

### CODE REVIEW

File: ch_tools/monrun_checks/ch_orphaned_objects.py [hunk 1]
```diff
@@ -52,7 +56,7 @@ def orphaned_objects_command(
         return Result(CRIT, str(e))
 
     total_size = state.orphaned_objects_size
-    error_msg = state.error_msg
+    error_msg = _error_message_format(state.error_msg)
```
Comment (aalexfvk) [inline ch_tools/monrun_checks/ch_orphaned_objects.py:59]: Consider putting this action [there](https://github.com/yandex/ch-tools/blob/35d396c5a46f143c0168d81eecafef25296aac9a/ch_tools/common/result.py#L46) to make it more general

Comment (NikitaUnisikhin) [reply ch_tools/monrun_checks/ch_orphaned_objects.py]: thks, that's much better. done

File: tests/features/monrun.feature [hunk 2]
```diff
@@ -513,3 +513,14 @@ Feature: ch-monitoring tool
...
+  
+  Scenario: Check clickhouse orphaned objects with long error_msg
+    When we create file /tmp/object_storage_cleanup_state.json with data "{ \"orphaned_objects_size\": 0,  \"error_msg\": \"Code: 27. DB::Exception: Cannot parse: input:: expected '\\\\t' before: 'klg%2D1acvr8hmq0n16qm5%2Edb%2Eyandex%2Enet\\\\ndefault\\\\n6736d483-516a-4892-87d4-084d5c1f6d3c\\\\n': While executing SystemRemoteDataPaths. (CANNOT_PARSE_INPUT_ASSERTION_FAILED) (version 24.8.5.115 (official build))  Query: SELECT obj_path, obj_size FROM _system.listing_objects_from_object_storage AS object_storage LEFT ANTI JOIN remoteSecure('klg-1acvr8hmq0n16qm5.db.yandex.net', system.remote_data_paths) AS object_table ON object_table.remote_path = object_storage.obj_path AND object_table.disk_name = 'object_storage' SETTINGS traverse_shadow_remote_data_paths=1 FORMAT TabSeparated (klg-1acvr8hmq0n16qm5.mdb.yandex.net)\" }"
+    And we execute command on clickhouse01
+    """
+    ch-monitoring orphaned-objects --state-local
+    """
+    Then we get response
+    """
+    2;Code: 27. DB::Exception: ... (CANNOT_PARSE_INPUT_ASSERTION_FAILED) (version 24.8.5.115 (official build)) ...
```
Comment (aalexfvk) [inline tests/features/monrun.feature:525]: Let's use `we get response matches` for checking response without specific versions

Comment (NikitaUnisikhin) [reply tests/features/monrun.feature:525]: The purpose of this test is to show the final error message, we are parsing an error that already contains "(version 24.8.5.115 (official build))", that is, this result of this test does not depend on the version of clickhouse.

Maybe I didn’t understand why to change this test.

Comment (aalexfvk) [reply tests/features/monrun.feature:525]: Ok, I got confused with the `Scenario: Check Geobase` test. But it is already fixed

**Review by aalexfvk — APPROVED** [review]

**Pull request merged**